packages/taco/package.json

```
@@ -32,7 +32,7 @@
     "clean": "rm -rf dist",
     "exports:lint": "ts-unused-exports tsconfig.json --ignoreFiles src/index.ts",
     "generate-zod-docs": "pnpm dlx tsx scripts/schema-docs-generation.ts",
-    "integration-test": "vitest run --config integration-test/vitest.config.ts",
+    "integration-test": "export RUNNING_IN_CI=true && vitest run --config integration-test/vitest.config.ts",
```

One of the reasons for not doing this previously is that this causes the test suite to take much longer. Not everyone wants to run the integration test every time and we should give devs the option. I think that anyone that wants to run it locally can explicitly set the environment variable themselves.

The environment variable is set for CI.

```suggestion
    "integration-test": "vitest run --config integration-test/vitest.config.ts",
```

---

The `export var=value` is limited to the only script/command using it. So, this is limited to `integration-test` which is not called by the script `test` and it does not cause any change in the current behaviour of running `pnpm test`.

---

Oh. Ok. I thought it changed the behaviour of running `pnpm test`. Ignore me then.